src/components/WeatherData.js

```diff
@@ -0,0 +1,9 @@
+const GetWeatherData = async (location, unit = 'metric') => {
+  const weather = await fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${location}&appid=87ca226fc9777f45c599892ee088e23b&units=${unit}`,
```

Comment:
Let's look into the project specification, as we both know, all other students are using this too;
so, we need to;
1. make different requests for the Celsius data and the Fahrenheit data.
2. add error handling for promises or async/await

![image](https://user-images.githubusercontent.com/31897434/81986015-06cdc780-962f-11ea-9fe5-60a32764c8db.png)